sdk/spring-sdk/src/it/java/com/example/wiring/eventsourcedentities/counter/CreateEmployee.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.eventsourcedentities.counter;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CreateEmployee {
+  public final String name;
+  public final String email;
+
+  @JsonCreator
+  public CreateEmployee(@JsonProperty String name, @JsonProperty String email) {
+    this.name = name;
+    this.email = email;
+  }
+}
```

Comment:
I don't think this one is being used for the IT tests, is it?

Reply:
Good catc. No, it's not. I will remove it